register-service/src/main/resources/sql/20211118_05_create_fitness_class_table.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE `fitness_class`
```

Comment:
This is need conversation on daily. Is the fitness_class?  It is strange for me. What is it?